CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(ros_workspace)
 
 find_package(ament_cmake_core REQUIRED)
 
-set(PYTHON_INSTALL_DIR "lib/python3.5/site-packages")
+set(PYTHON_INSTALL_DIR "lib/python3.6/site-packages")
```

Comment:
We should be able to get this from python directly. e.g. (not saying it's the best way..) getting the version by calling `execute_process` with something like:
`${PYTHON_EXECUTABLE} -c from distutils import sysconfig; sysconfig.get_python_version()` 

On my xenial machine: 
```
$ python3 -c "from distutils import sysconfig; print(sysconfig.get_python_version())"
3.5
```

Reply:
This looks good. I'll give this a try.